ldk/javascript/examples/javascript-loop/index.js

```diff
@@ -0,0 +1,18 @@
+import { clipboard, whisper } from '@oliveai/ldk'
+
+function clipboardListenAndWhisper() {
+    clipboard.listen(true, (incomingText) => {
+        whisper.create({
+            label: 'Clipboard Text Whisper',
```

Comment:
You may want to provide instructions here on what user should do and what they could see. In case if someone will try to run it, is it clear on what they need to do? Just a suggestion.

Reply:
We can iterate on this documentation as LDK authors request more clarification.